test/integration/consul-container/upgrade/healthcheck_test.go

```diff
@@ -0,0 +1,328 @@
+package consul_container
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+
+	consulCluster "github.com/hashicorp/consul/integration/consul-container/libs/consul-cluster"
+	consulNode "github.com/hashicorp/consul/integration/consul-container/libs/consul-node"
+
+	"github.com/hashicorp/consul/integration/consul-container/libs/utils"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+
+	"github.com/stretchr/testify/require"
+)
+
+var curImage = flag.String("uut-version", "local", "docker image to be used as UUT (unit under test)")
```

Comment:
Could we be more explicit in this flag name? UUT wasn't immediately obvious to me without the explanatory text, but `target` or `current` probably would be?

```suggestion
var curImage = flag.String("target-version", "local", "docker image to be used as test target")
```